### ISSUE TITLE
Storage of transformation information on training data

### ISSUE THREAD

**Issue by MichalStachowicz** [kedro-org/kedro#1984]
## Description
In my project I would like to have a pipeline using a pre-trained model that is on MLflow. The problem I am facing is that I don't know how to convey the transformation information on the data in the preprocessing. For example, for normalisation we need a minimum and a maximum value to be able to transform the test set in the same way. Is there a solution to this issue? Do you recommend using sklearn Pipelines ?

**Comment by noklam** [comment]
@MichalStachowicz 
This is a good open-end question to ask in our Slack community. We don't have recommendations about a specific framework.   

Join Slack: https://join.slack.com/t/kedro-org/shared_invite/zt-1eicp0iw6-nkBvDlfAYb1AUJV7DgBIvw

Closing this issue.
